lib/modules/dosomething/dosomething_signup/dosomething_signup.query.inc

```diff
@@ -41,10 +41,10 @@ function dosomething_signup_get_signups_query($params) {
  */
 function dosomething_signup_build_signups_query($params = []) {
   $query = db_select('dosomething_signup', 's');
-  $query->leftJoin('dosomething_reportback', 'rb', 's.nid = rb.nid AND s.run_nid = rb.run_nid');
+  $query->leftJoin('dosomething_reportback', 'rb', 's.uid = rb.uid and s.nid = rb.nid'); 
   $query->fields('s', ['sid', 'uid', 'nid', 'run_nid', 'timestamp']);
   $query->fields('rb', ['rbid']);
-  $query->where('s.uid = rb.uid');
+  $query->orderBy('rb.rbid', 'DESC');
```

Comment:
I don't think we actually need the orderBy in this query, that was just for when we were testing